### PR TITLE
CC-29918 Fixed AddressForm's city field input type.

### PR DESCRIPTION
**Pull request by AsonUnique**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/CC-29918

### CODE REVIEW

**Comment by AsonUnique** [comment]
Added `package-lock.json` update to the PR, because CI was failing otherwise.
As I can see, it was updated also in https://github.com/spryker-shop/b2b-demo-marketplace/pull/151 so we can just wait until it'll be merged.

**Review by DmitryLymarenko — APPROVED** [review]

**Review by DmitryLymarenko — APPROVED** [review]

**Pull request merged**
